src/libraries/Common/src/System/Security/Cryptography/SlhDsa.cs

```diff
@@ -93,23 +93,21 @@ public void Dispose()
         }
 
         /// <summary>
-        ///   Sign the specified data, writing the signature into the provided buffer.
+        ///   Signs the specified data, writing the signature into the provided buffer.
         /// </summary>
         /// <param name="data">
         ///   The data to sign.
         /// </param>
         /// <param name="destination">
-        ///   The buffer to receive the signature.
+        ///   The buffer to receive the signature. Its length must be exactly
```

Comment:
Moved this up from remarks since [guidelines](https://github.com/dotnet/dotnet-api-docs/wiki/Parameters) say:

> Include important information such as which enumeration values are supported, whether the value can be null, the unit of measurement